app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+from __future__ import annotations
```

Comment:
this import not needed here

Reply:
without from __future__ import annotations
  I have this error
   def feed_animals(animals_list: list[Animal]) -> int:
E   TypeError: 'type' object is not subscriptable